modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java

```diff
@@ -64,19 +65,41 @@ public static void initDatabase(String dbDirectory, String dbName, String sqlFil
         } catch (ClassNotFoundException | SQLException e) {
             //Do nothing
         } finally {
-            try {
-                if (st != null) {
-                    st.close();
-                }
-                if (connection != null) {
-                    connection.close();
-                }
-            } catch (SQLException e) {
-                //Do nothing
+            releaseResources(connection, st);
+        }
+    }
+
+    /**
+     * Create H2 DB with the given name and initialize with given SQL file.
+     *
+     * @param dbDirectory Name of the DB directory.
+     * @param dbName  Name of the DB instance.
+     * @param sqlFile SQL statements for initialization.
+     */
+    public static void initH2Database(String dbDirectory, String dbName, String sqlFile) {
+        Connection connection = null;
+        Statement st = null;
+        try {
+            Class.forName("org.h2.Driver");
+            String jdbcURL = "jdbc:h2:file:" + dbDirectory + dbName;
+            connection = DriverManager.getConnection(jdbcURL, "sa", "");
+            String sql = readFileToString(sqlFile);
+            String[] sqlQuery = sql.trim().split("/");
+            Statement st1 = connection.createStatement();
```

Comment:
Shall we rename st1 variable name to a meaningful name?

Reply:
Fixed.